src/reports/routes/reportsRoute.js

```diff
@@ -13,5 +13,6 @@ router.get('/:test_id/reports/', swaggerValidator.validate, reports.getReports);
 router.get('/last_reports/', swaggerValidator.validate, reports.getLastReports);
 router.post('/:test_id/reports/', swaggerValidator.validate, reports.postReport);
 router.post('/:test_id/reports/:report_id/stats', swaggerValidator.validate, reports.postStats);
-
+router.get('/:test_id/reports/:report_id/export/:file_format',swaggerValidator.validate,reports.getExportedReport);
+router.get('/reports/compare/export/:file_format',swaggerValidator.validate,reports.getExportedCompareReport);
```

Comment:
i know it's not REST but the how this api is configured now is little bit too open
because the order of test id and report id is important (it must match)
i would consider even changing this to POST api with body of pairs